grpc-gateway/service/updateResources_test.go

```diff
@@ -21,18 +21,18 @@ import (
 func TestRequestHandler_UpdateResourcesValues(t *testing.T) {
 	deviceID := test.MustFindDeviceByName(test.TestDeviceName)
 	type args struct {
-		req pb.UpdateResourceValuesRequest
+		req pb.UpdateResourceRequest
```

Comment:
What led you to renaming this ?

Reply:
DeleteResource/CreateResource/UpdateResource ....